types/parse/index.d.ts

```diff
@@ -675,6 +676,13 @@ declare namespace Parse {
             // Sort documentation https://docs.mongodb.com/v3.2/reference/operator/aggregation/sort/#pipe._S_sort
             sort?: {[key: string]: 1|-1};
         }
+
+        // According to https://parseplatform.org/Parse-SDK-JS/api/2.1.0/Parse.Query.html#fullText
+        interface FullTextOptions {
+          language:	string;
```

Comment:
Are these all required?  Frequently, options types have optional members.

Reply:
Ping @ChrisGrant

Reply:
The docs agree; one of the examples is `query.fullText('field', 'term', { language: 'es', diacriticSensitive: true });`.  @ChrisGrant you wanna add ` ?` to the end of each of these props for correctness?